src/core/xref.js

```diff
@@ -754,7 +757,10 @@ class XRef {
   }
 
   get lastXRefStreamPos() {
-    return this._xrefStms.size > 0 ? Math.max(...this._xrefStms) : null;
+    return (
+      this.#firstXRefStmPos ||
```

Comment:
Should we respect this entry if it's zero, since it's currently ignored then, or is that not correct and/or can't happen?
I.e. should the `||` be possibly be replaced with `??` here?